bokehjs/src/coffee/api/typings/models/transforms.d.ts

```diff
@@ -0,0 +1,4 @@
+declare namespace Bokeh {
+  export interface Transform extends Model, ITransform {}
+  export interface ITransform extends IModel {}
+}
```

Comment:
@mattpap these typings changes compile without error, but please check them. I only added the transform base, I don't have enough TS experience to add the rest.